app/views/pregnancies/_patient_information.html.erb

```diff
@@ -77,7 +77,53 @@
                        options_for_select(referred_by_options,
                                           pregnancy.referred_by),
                                           autocomplete: 'off' %>
-          <%= f.text_field :special_circumstances, autocomplete: 'Off' %>
+          <div>       
+            <%= f.label :special_circumstances, 'Special circumstances' %>   
+            <br />
+            <div>
+              <div class="col-sm-6">
+                <div>
+                  <%= check_box_tag('pregnancy[special_circumstances][]', 'Fetal anomaly', 
```

Comment:
I think this is outstanding work and is super close! 

One of the few nonstandard tools we use is the `rails_bootstrap_forms` gem, which makes stuff like this a bit easier. I'd check out this https://github.com/bootstrap-ruby/rails-bootstrap-forms#checkboxes-and-radios and the section on `form_group`. You might try something like:

```ruby
f.form_group :special_circumstances, label: 'Special circumstances' do
  div class='col-sm-6'
    f.check_box 'Homelessness', checked: pregnancy.special_circumstances.include?('Homelessness')
```

etc. that'll take care a lot of the kind of pain-in-the-ass styling for you.